usr.sbin/fdwrite/fdwrite.c

```diff
@@ -26,7 +25,7 @@ format_track(int fd, int cyl, int secs, int head, int rate,
      int gaplen, int secsize, int fill, int interleave)
 {
     struct fd_formb f;
-    register int i,j;
+    int i,j;
     int il[100];
 
     memset(il,0,sizeof il);
```

Comment:
while you're already here, why not format this according to `style(9)`.

Reply:
I intentionally left it behind as `style(9)` also says to indent the first level with 8 characters (tabs) and running `indent(1)` drastically changes the indentation (which I wasn't sure about whether should I apply or not)...

I think for now I'll apply it for these (CI) warnings only. If full indentation is acceptable, then I will add that too.
CI Link: https://github.com/freebsd/freebsd-src/actions/runs/7517809483/job/20464384723?pr=1059